sqlmesh/core/context.py

```diff
@@ -793,6 +794,8 @@ def table_diff(
             target = next(
                 snapshot for snapshot in target_env.snapshots if snapshot.name == model.name
             ).table_name()
+            source_alias = source_env.name
```

Comment:
what's the point of source_alias? i'm not sure i understand what this is

Reply:
because without it, you get ugly snapshot names using the syntax I prescribed. 
it is better to capture the environment name to connect the dots or its extremely non-obvious

Reply:
see the user experience here thanks to the alias, otherwise those snapshot names mean nothing other than the assumption the left/top is source and right/bot is target

<img width="1040" alt="image" src="https://github.com/TobikoData/sqlmesh/assets/41213451/34a1c1ed-9903-4790-a514-ebe055f11d6a">

Reply:
Following the exact same logic,

<img width="1043" alt="image" src="https://github.com/TobikoData/sqlmesh/assets/41213451/57504d77-9f56-4226-8454-19b6ada5aca3">

I _could_ possibly make the case that the `s__{{column}}` could be better represented to the end user as `{{env-name}}__{{column}}` provided the env name was available and its not a direct diff of 2 tables/snapshots. (see above, not committed to this PR). 

Not a big deal really but something to consider. Its all about UX in this case